src/scenes/home/opCodeCon/opCodeCon.css

```diff
@@ -1,37 +1,40 @@
 .hero {
-  background-image: url('./raleigh.jpg');
+  background-image: url('./OpCodeCon_graphics.png');
+  background-repeat: no-repeat;
+  background-position: center;
 }
 
-@media screen and (min-width: 1200px) {
+@media screen and (min-width: 1100px) {
   .hero {
     background-size: 100%;
   }
 }
 
-@media screen and (max-width: 768px) {
+@media screen and (min-width: 500px) and (max-width: 1100px) {
+  .hero {
+    background-size: 80%;
+    padding-top: 20%;
+    background-position: top;
+  }
+}
+
+@media screen and (max-width: 480px) {
   .hero {
-    background-size: auto 150%;
+    background-size: 100%;
+    padding-top: 115px;
+    background-position: top;
   }
 }
 
 .heading {
-  color: #47566b;
-  padding: 3rem;
+  color: #249cbc;
+  text-align: center;
+  padding-top: 25%;
   height: 600px;
 }
 
 @media screen and (max-width: 768px) {
-.heading {
+  .heading {
     height: 450px;
   }
 }
```

Comment:
The media queries for the hero background image and the heading text need some adjustment at screen sizes around 1366px width - the header text overlays the text in the background image on my laptop screen: 

![selection_050](https://user-images.githubusercontent.com/8835055/40285348-e248a01c-5c68-11e8-94f9-bdec79ce7fb5.png)

@Wimsy113 I'm thinking the 'OpCodeCon / Raleigh, NC / Sept 19-20' text would be easier to position/resize via CSS if it was in the HTML markup. That would also better from an accessibility standpoint.

Is there some reason that text is embedded in the background image?

We could just use the image `Opcodecon_graphic.png` which doesn't include the text.

Reply:
Yea the reason why the text is embedded in the image is because I was having trouble with resizing the text via CSS,it was easier to have the text as part of the graphic, I was actually working on some other images because there's sort of a ratio I'm trying to keep between the image and the words.